docs/cloud-native-security/cspm-findings.asciidoc

```diff
@@ -21,10 +21,12 @@ By default, the Findings page lists all findings, without grouping or filtering.
 [discrete]
 === Group findings 
 
-. Click **Group findings by:** to open a list of fields by which you can group findings. Select one of the suggested group-by fields, or click **Custom field** to choose your own group-by field. 
-. When grouping is turned on, click a group to expand it and examine all findings within that group.
+. Click **Group findings by:** to open a list of fields by which you can group your data. Select one of the suggested fields, or click **Custom field** to choose your own. You can select up to three group-by fields at once. 
```

Comment:
This step appears a few other times in the PR, so the same suggested edits apply there too.
```suggestion
. Click **Group findings by** to group your data by a field. Select one of the suggested fields or **Custom field** to choose your own. You can select up to three group-by fields at once. 
```